File: C/Exercicio 25.c
#include<stdlib.h>
#include<stdio.h>
	
	//Função principal do programa
	int main(int argc, char const *argv[])
	{
	// Declaração das variáveis
	int num, res, cont;
	
	//25° Programa - Exibir a tabuada do número cinco no intervalo de um a dez
	                      
	//Exibe na tela 
	printf("Exibir a Tabuada do Numero Cinco no intervalo de um a Dez");
	
	//Solicita entrada de dados para o usuário e grava o valor digitado na variável "num"
	printf("\nDigite o Numero Cinco Para a Tabuada: ");
	scanf("%i",&num);
	
	//Comando While, usado sem intervalos definidos
	while(num!=5){
	printf("Erro!\nApenas o Numero Cinco. Digite Novamente: ");
	scanf("%i",&num);
	}
	
	//Comando for, usado em intervalos definidos
	for (cont=0; cont<=10; cont++) {
	res = num * cont;   
	printf("%i x %i = %i\n", num, cont, res);
	}
	
	return 0;
	}

File: Python/Exercicio25.py
	print("Exibir a tabuada do 5 no intervalo de 1 a 10")
	
	num = int(input('Digite o número 5 para a tabuada: '))
	
	while num !=5:
	    print("Erro!Apenas o número 5")
	    num = int(input("Digite novamente: "))
	
	for i in range(1, 11, 1): #range ( parte do 1, vai até inteiro menor que 11, no passo 1 em 1)  
	    res = num * i
	    print('{num} x {i} = {res}')
